cypress/e2e/ui-tests/test-project-contributor.cy.js

```diff
@@ -0,0 +1,93 @@
+describe("Test project contributor user functionality", () => {
+
+    // TODO: share logged in session across tests
+
+    // username here must match username in db-seed-contributor.sh
+    const username = "e2e_tests_contributor_tester"
+    const pwd = "test12345"
```

Comment:
May be we could put this in some .json or .yaml config in order to decrease the amount of duplicate hardcoded values?

Reply:
Good point. I have moved it to test fixtures in the coming version.